### PR TITLE
Wonka-Conference : Add Home and About Pages - Mobile & Desktop Version

### PR DESCRIPTION
**Pull request by OmarRamoun**
## Changes Done :recycle:

A Conference Website for the Wonka Conference for family health. Built from scratch using HTML5, CSS3, JS, and mostly the Bootstrap framework, taking into account the requirements specified below. Link to the video description here.

### Link to Loom Video
- [link](https://www.loom.com/share/a65e01ea771447edb23a07935a095f23)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,28 +2,27 @@
 
 <!-- Feel Free to Add, Update, Delete Any Section you find needs so -->
 
-# Project Name
+# Wonka Conference Website
 
-> One paragraph statement about the project.
+> Wonka Converence 2021 Made with Vanilla HTML5, CSS3, & JS. The Website Contains A Homepage With Information About The Main Program Of The Conference Speakers And Partners And An About Page With Information About The Conference.
 
 ![screenshot](images/app_screenshot.png)
```
Comment (emmanuellekamwa) [inline README.md:10]: - [x] Hello. Please this screenshot is not necessary here. You can choose to either replace it with an actual screenshot of your project or you can just delete it.
- [x] I have noticed that you have forgotten to add the link to your video presentation. Please add it.

File: README.md [hunk 2]
```diff
@@ -87,27 +84,20 @@ To get a local copy up and running follow these simple example steps.
...
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
 - Inspiration
-- etc
+- My Passion for Programming
```
Comment (emmanuellekamwa) [inline README.md:100]: - [x] In this section, don't forget to add credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  </section>
+
+  <footer class="footer px-2 py-4 d-flex gap-3 align-items-center">
+    <img class="w-20 flex-1" src="../images/cc.svg" alt="">
+    <div class="copyrights flex-grow-3 flex-shrink-3 w-50">
+      <p class="mb-0">
+        2021 WONCA World Conference. All rights reserved.
+      </p>
+    </div>
+  </footer>
```
Comment (emmanuellekamwa) [inline about/index.html:139]: - [x] The footer contains a logo aligned to the left and text aligned to the right but on your page, the logo and the text are aligned on the left. Please fix that on both pages.

File: index.html [hunk 4]
```diff
@@ -6,15 +6,181 @@
...
+          <p class="text-light mb-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt incidunt debitis.
+          </p>
+        </div>
+      </div>
+      <div class="footer d-flex mt-5">
+        <button type="button" class="btn btn-custom text-light mx-auto p-3 bg-accent">Join Conference 2021</button>
+        <a class="d-none text-center" href="#">Learn More Information</a>
+      </div>
+    </div>
+  </section>
```
Comment (emmanuellekamwa) [inline index.html:144]: - [x] In mobile view, the icons of this section are too big. Please try and reduce their sizes.

File: index.html [hunk 5]
```diff
@@ -6,15 +6,181 @@
...
+          </div>
+          <div class="col-right col-sm-12 col-md-12 col-lg-6 d-flex flex-column gap-3">
+          </div>
+        </div>
+      </div>
+      <div class="footer d-md-none d-flex mt-5">
+        <a class="btn btn-custom text-capitalize text-light mx-auto p-3 bg-accent">more speakers</a>
+      </div>
+    </div>
+  </section>
```
Comment (emmanuellekamwa) [inline index.html:161]: - [x] On this section, I suggest you should remove the borders around the divs and add some background behind the images.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛠🛑**
Hello @MrRamoun 👋🏽
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._👇🏽👀

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by george-shammar — APPROVED** [review]
Hi @MrRamoun , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

:heavy_check_mark: This is truly impressive and thank you for making the required changes. :+1: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**